tests/positive/partitions/complex.go

```diff
@@ -43,8 +43,8 @@ func KitchenSink() types.Test {
 					"number": 1,
 					"start": 2048,
 					"size": 65536,
-					"typeGuid": "316f19f9-9e0f-431e-859e-ae6908dbe8ca",
-					"guid": "53f2e871-f468-437c-b90d-f3c6409df81a",
```

Comment:
There are a lot of guids/type guids that used to change but now do not. Can you make sure all partitions that used to change guids/typeguids between the input and output still do?